sdk/src/client/stronghold/error.rs

```diff
@@ -26,10 +26,30 @@ pub enum Error {
     #[error("no mnemonic has been stored into the Stronghold vault")]
     MnemonicMissing,
     /// Procedure execution error from Stronghold
-    #[error("Stronghold reported a procedure error: {0}")]
+    #[error("stronghold reported a procedure error: {0}")]
     Procedure(#[from] iota_stronghold::procedures::ProcedureError),
     // TODO remove later
     /// Invalid mnemonic error
     #[error("invalid mnemonic {0}")]
     InvalidMnemonic(String),
+    /// Unsupported snapshot version
+    #[error("unsupported snapshot version, migration required")]
```

Comment:
does not include the variables in the message

Reply:
Done